src/methods/function.js

```diff
@@ -323,7 +323,7 @@ export class FunctionImplementation {
           break;
         case "SwitchStatement":
           for (let switchCase of ((ast: any): BabelNodeSwitchStatement).cases) {
-            statements = statements.concat(switchCase.consequent);
+            Array.prototype.push.apply(statements, switchCase.consequent);
```

Comment:
We support ES2015, why not use array spread? 

`statements.push(...switchCase. consequent);`

Reply:
Are they equivalent wrt performance or does the spread create an extra array?

Reply:
AFAIK they are equivalent if not faster, you should be able to tell with a quick sanity test locally on our bundle?